<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>View3DWidget</class>
    <widget class="QWidget" name="View3DWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>960</width>
                <height>640</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>3D Viewer</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="200,2,40">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QWidget" name="container" native="true"/>
            </item>
            <item>
                <spacer name="horizontalSpacer">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>5</width>
                            <height>20</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <layout class="QVBoxLayout" name="verticalLayout" stretch="1,7,2">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <item>
                        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QToolButton" name="reset_btn">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">background-color: rgb(78, 154, 6);
                                            color: rgb(255, 255, 255);
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>reset</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="pick_layout" stretch="1,9,2">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QLabel" name="pick_label">
                                    <property name="font">
                                        <font>
                                            <pointsize>16</pointsize>
                                        </font>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">color: rgb(0, 0, 0);
                                            background-color: rgb(255, 255, 255);
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>selected</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QTextEdit" name="pick_edit">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="inputMethodHints">
                                        <set>Qt::ImhNone</set>
                                    </property>
                                    <property name="readOnly">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <layout class="QHBoxLayout" name="expand_layout" stretch="1">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QToolButton" name="expand_btn">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true">background-color: rgb(196, 160, 0);
                                                    color: rgb(255, 255, 255);
                                                </string>
                                            </property>
                                            <property name="text">
                                                <string>expand</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QVBoxLayout" name="format_layout" stretch="1,2,2">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <item>
                                <widget class="QLabel" name="format_label">
                                    <property name="styleSheet">
                                        <string notr="true">color: rgb(0, 0, 0);
                                            background-color: rgb(255, 255, 255);
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>format</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignCenter</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QComboBox" name="format_box">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="focusPolicy">
                                        <enum>Qt::NoFocus</enum>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">QComboBox {
                                            border : 2px solid gray ;
                                            border-radius: 4px ;
                                            padding : 1px 2px 1px 20px ;
                                            }
                                        </string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QToolButton" name="format_btn">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="styleSheet">
                                        <string notr="true">background-color: rgb(32, 74, 135);
                                            color: rgb(255, 255, 255);
                                        </string>
                                    </property>
                                    <property name="text">
                                        <string>export</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
